### ISSUE TITLE
Use full content width by default

### ISSUE THREAD

**Issue by Zethson** [MrDogeBro/sphinx_rtd_dark_mode#10]
Hey,

I was waiting for this forever and I hope that you will maintain this!

Anyways, 

```
.wy-nav-content {
    max-width: none;
}
```

is always required as custom css to use the full width of the screen. What do you think about making this a default?

**Comment by MrDogeBro** [comment]
Could you show a picture of the difference? This extension is for dark mode tho, and therefore if its not related to theming, it may not be added. We would have to see.

**Comment by Zethson** [comment]
It goes slightly beyond theming, so I guess not. I just patch my own css

**Comment by MrDogeBro** [comment]
Ok, could you show a picture tho? There is a potential that it could always just be added as an extra option that can be enabled in the config.

**Comment by Zethson** [comment]
@MrDogeBro look at this for example: https://sc-toolbox.readthedocs.io/en/latest/installation.html

It uses the full width of the screen. The default of the theme only uses like 400 pixels or something (I forgot)

**Comment by MrDogeBro** [comment]
Yeah I see — thats what I thought you were talking about. What do you think about just a bool config option for toggling in (off by default)?

**Comment by Zethson** [comment]
Yeah sure, why not.
